### ISSUE TITLE
Adding {{liquid-modal}} to application.hbs causing warning

### ISSUE THREAD

**Issue by chrism** [ef4/liquid-fire#164]
Hi,

I've noticed after adding {{liquid-modal}} while using HTMLBars & canary i'm now getting this warning.

````
 WARNING: You're attempting to render a view by passing boundContextBinding to a view helper without a quoted value, but this syntax is ambiguous. You should either surround boundContextBinding's value in quotes or remove `Binding` from boundContextBinding
````

I'm using the master branch on liquid-fire

**Comment by minusfive** [comment]
Same with {{#liquid-with}}

**Comment by ef4** [comment]
These are fixed in 0.17.1 for use with Ember 1.10 and below and fixed in 0.18.0 for use with newer Embers.
